### DISCUSSION TITLE
DDS Status Reporting

### DISCUSSION THREAD

**Discussion by cappelaere** [opendcs/opendcs#164] [Q&A]
I have been looking at the xml status returned by DDS to support a Prometheus endpoint.  This is not really documented that I could find.  So I am guessing here.  Let me know if I am close.
It seems like that status returns, among other things, archive and recv quality for 24 time slots.  I am assuming that the hour attribute is the UTC hour of the day.  Hour=6 would mean the aggregated results for hour from 5am to 6am?
The html page seems to return the status from the last 5 hours + current partial status for current hour.
Since the xml status returns values for 24 hours, it might seem that results beyond current UTC hour might be from previous day???? Is that right?

**Comment by MikeNeilson** [comment]
I think that's all correct. 
But I'd need to dig into the code more to know for sure.
Will likely be a few days or more though, was doing that at 4am since I was awake anyways but my 8month old suddenly shifted to not sleeping peacefully through the night which has thrown off my DDS documentation efforts. https://github.com/opendcs/opendcs/pull/151, if you want to help by submitting PRs to that branch if you verify anything. Right now I'm just at the table of commands/responses and some baseline information. But I'll start on the status next when I get back to it.
